src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1377,6 +1376,8 @@ void AutomationInstrumentClipView::editPadAction(bool state, uint8_t yDisplay, u
 			}
 
 			if (firstPadX != 255 && firstPadY != 255 && firstPadX != xDisplay) {
```

Comment:
record the second press in a long press into the editPadPress struct so that you it will remain active if you let go of the first press.

this enables you to enter a very long continuing sequence of long presses